www/gatsby-node.js

```diff
@@ -15,6 +15,14 @@ const packages = require(`./src/utils/node/packages.js`)
 const features = require(`./src/utils/node/features.js`)
 const sections = [docs, showcase, starters, creators, packages, features]
 
+exports.createSchemaCustomization = async helpers => {
+  await Promise.all(
+    sections
+      .filter(section => typeof section.createSchemaCustomization === `function`)
+      .map(section => section.createSchemaCustomization(helpers))
+  )
+}
```

Comment:
[nit] I don't see much benefit in creating a loop for it. I rather have this explicit 😛  
```suggestion
  await docs.createSchemaCustomization(helpers)
}
```

Reply:
I didn't want someone to hit the same thing I did, haha. e.g. add an API and have it not work like you'd expect, it's super undiscoverable!

Reply:
oh because we split everything up? Sounds reasonable :)

Reply:
(Yeah I don't love that we _do_ split everything up, but oh well... )